math/mathcore/inc/TMath.h

```diff
@@ -528,9 +521,9 @@ struct Limits {
    template <typename Iterator> Double_t RMS(Iterator first, Iterator last);
```

Comment:
Should not add also here the documentation for RMS and StdDev and median ? 
For example what is the work array in median ?